### PR TITLE
Add Triangle tool

### PR DESCRIPTION
**Pull request by jwbmartin**
Fixes pretty much nothing

But it does add a triangle drawing tool
# Review Checklist
- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Did you deploy a staging branch?
## Optional
- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [x] If changes are made to the classifier, does the dev classifier still work?

### CODE REVIEW

**Comment by wgranger** [comment]
Looks good, but this should be added to the Dev Classifier before it is merged.

File: app/classifier/drawing-tools/triangle.cjsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    y: -1 * (@props.mark.r + (BUFFER / @props.scale.vertical)) * Math.sin theta
+
+  computePoints: (r) ->
+    cx = r 
+    cy = r 
+    
+    ax = cx * Math.cos( 240 )  - ( cy * Math.sin( 240 ) )
+    ay = cx * Math.sin( 240 )  + ( cy * Math.cos( 240 ) )
+    bx = cx * Math.cos( 120 )  - ( cy * Math.sin( 120 ) )
+    _by = cx * Math.sin( 120 ) + ( cy * Math.cos( 120 ) )
```
Comment (jwbmartin) [inline app/classifier/drawing-tools/triangle.cjsx]: I think "by" is an operator in coffeescript so it was causing issues.

Comment (wgranger) [inline app/classifier/drawing-tools/triangle.cjsx]: Ahh. Interesting.

**Comment by wgranger** [comment]
Alllright. 👏

**Comment by jwbmartin** [comment]
tumblr_nuvcw1wrfa1utn1fuo1_500

**Pull request merged**